src/Styles/foundation/_text-decoration.scss

```diff
@@ -2,6 +2,7 @@
 .strike           { text-decoration: line-through; }
 .underline        { text-decoration: underline; }
 .underline-dotted { text-decoration: underline; text-decoration-style: dotted }
+.required span.label-span:first-child::after { content:" *"; color: red; }
```

Comment:
```suggestion
.required .label-span:first-child::after { content:" *"; color: red; }
```
the element is required. it does make the query slightly faster but at the expense of flexibility.